lib/kitchen/provisioner/chef_base.rb

```diff
@@ -47,6 +47,13 @@ class ChefBase < Base
 
       default_config :require_chef_omnibus, true
       default_config :chef_omnibus_url, "https://omnitruck.chef.io/install.sh"
+      # `chef_omnibus_cache` is an option that will let us be specific with the
+      # directory we want omnibus packages to be cached, it will be useful for
+      # other drivers that can mount or share a folder to avoid downloading the
+      # same software every converge
+      default_config :chef_omnibus_cache do |provisioner|
```

Comment:
provisioners outside of Chef _could_ take advantage of this paradigm.  Should we rename it (and the internal path) to something more genertic? Maybe `:provisioner_cache` and `/tmp/provisioner/cache`? Then, the Chef provisioner itself could leverage the provisioner cache in its own way.